ohos/libpag/src/main/ets/PAGLayer.ets

```diff
@@ -80,7 +80,10 @@ export class PAGLayer {
   }
 
   /**
-   * The final matrix for displaying, it is the combination of the matrix property and current matrix from animation.
+   * The final matrix for displaying, it is the combination of the matrix property and current
```

Comment:
Returns the layer's display matrix by combining its matrix() property with the current animation matrix from the AE timeline. This does not include the parent layer's matrix. To calculate the final bounds relative to the PAGSurface, use the PAGPlayer::getBounds(PAGLayer layer) method.